.circleci/config.yml

```diff
@@ -76,18 +75,15 @@ workflows:
   version: 2
   build:
     jobs:
-      - "python-3.7-symengine"
-      - "python-3.7-sympy-1.3"
-      - "python-3.7-sympy-1.4"
-      - python-3.7:
-          filters:
-            tags:
-              only: /^v.*$/
```

Comment:
I think I might have misunderstood this bit - I assume it meant "run only on tags", but I think this means "run on tasg as well"?

Reply:
Assuming I was wrong, and restored the old version

Reply:
Yes, this is very tricky. When you see a target with config `tags` and `only`, it's counter-intuitive that it actually also runs for branches.

See my commit https://github.com/pygae/galgebra/commit/3070d117467efd489c4696c75e1086f19a8a3239#diff-1d37e48f9ceff6d8030570cd36286a61 .

The story (of my troubleshooting) is that:

- I need to run `publish` for no branch and for tags starting with `v`
- `publish` requires `python-3.7`
- it turns out that since I didn't explicitly ask CircleCI to run `python-3.7` for tags starting with `v`, `publish` is not running as well so I have to add it explicitly.

The root cause of this counter-intuitive configuration is that CircleCI runs for all branches and no tags by default, it's like blacklisting for the former and whitelisting for the latter. So my configuration specifies no blacklist for the former and a whitelist for the latter, thus CircleCI runs for both.

Reply:
As far as I can tell, your final config is correct, we'll know when we publish the next release.